### PR TITLE
opengl: link to OpenGLES framework on iOS & tvOS

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

On iOS & tvOS, we have to link to OpenGLES framework.
See https://developer.apple.com/documentation/opengles?language=objc

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by dmn-star — COMMENTED** [review]
Wasn't OpenGL ES deprecated?

**Comment by SpaceIm** [comment]
It is

File: recipes/opengl/all/conanfile.py [hunk 1]
```diff
@@ -74,6 +74,8 @@ def package_info(self):
         if self.settings.os == "Macos":
             self.cpp_info.defines.append("GL_SILENCE_DEPRECATION=1")
             self.cpp_info.frameworks.append("OpenGL")
+        elif self.settings.os in ["iOS", "tvOS"]:
+            self.cpp_info.frameworks.append("OpenGLES")
```
Comment (Croydon) [inline recipes/opengl/all/conanfile.py:78]: Shouldn't OpenGL ES get its own package?

Comment (SSE4) [reply recipes/opengl/all/conanfile.py:78]: I think so

Comment (SpaceIm) [reply recipes/opengl/all/conanfile.py:78]: yes I agree

**Pull request closed without merging**